src/main/java/org/zalando/jackson/datatype/money/MoneyModule.java

```diff
@@ -14,34 +14,75 @@
 
 public final class MoneyModule extends SimpleModule {
 
+    private final MonetaryAmountFactory<? extends MonetaryAmount> amountFactory;
+    private final MonetaryAmountFormatFactory formatFactory;
+    private final FieldNames names;
+
+    @SuppressWarnings("deprecation")
     public MoneyModule() {
         this(new MoneyFactory());
     }
 
+    /**
+     * @deprecated use {@link #withAmountFactory(MonetaryAmountFactory)}
+     * @param factory the amount factory used for deserialization of monetary amounts
+     */
+    @Deprecated
```

Comment:
Hoping for Java 9 - what about adding a hint since when and why those methods are deprecated?